### ISSUE TITLE
Java contracts fail to install on test-network

### ISSUE THREAD

**Issue by davidkel** [hyperledger/caliper-benchmarks#159]
There are 2 java contracts
samples/fabcar/java
api/fixed-asset/java

Both appear to require java 8 but when attempting to deploy both return with similar error messages

```
Compiling Java code...
~/caliper-benchmarks/src/fabric/samples/fabcar/java ~/fabric-samples/test-network
Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* What went wrong:
Task 'installDist' not found in root project 'java-chaincode-bootstrap'
```

```
Compiling Java code...
~/caliper-benchmarks/src/fabric/api/fixed-asset/java ~/fabric-samples/test-network

FAILURE: Build failed with an exception.

* What went wrong:
Task 'installDist' not found in root project 'fixed-asset'.
```

If I try java 11 then I get
```
* What went wrong:
Could not determine java version from '11.0.11'
```

**Comment by davidkel** [comment]
For fixed-asset the solution is
1. upgrade the version of gradle to 6.5.1
2. include the `application` plugin

gradle-wrapper.properties
```
distributionBase=GRADLE_USER_HOME
distributionPath=wrapper/dists
distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-bin.zip
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists
```

build.gradle
```
plugins {
    id 'com.github.johnrengelman.shadow' version '5.1.0'
    id 'application'
}

...

tasks.withType(JavaCompile) {
  options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation" << "-parameters"
}

installDist.dependsOn test
```
(changed java-->application in plugins, added the last line)

**Comment by davidkel** [comment]
initial java issues so that they can be deployed to test network are fixed by #180, other issues are covered by #181 
and #182
